==== stdout ====

==== stderr ====
[1m[32m    Finished[0m release [optimized] target(s) in 0.18s
[1m[32m     Running[0m `target/release/examples/asset_decompression`
[2m2023-11-20T18:58:56.377253Z[0m [32m INFO[0m [2mbevy_winit::system[0m[2m:[0m Creating new window "App" (0v0)
[2m2023-11-20T18:58:56.377455Z[0m [33m WARN[0m [2mlog[0m[2m:[0m XRandR reported that the display's 0mm in size, which is certifiably insane    
[2m2023-11-20T18:58:56.377510Z[0m [32m INFO[0m [2mlog[0m[2m:[0m Guessed window scale factor: 1    
[2m2023-11-20T18:58:56.524649Z[0m [32m INFO[0m [2mbevy_render::renderer[0m[2m:[0m AdapterInfo { name: "llvmpipe (LLVM 15.0.7, 256 bits)", vendor: 65541, device: 0, device_type: Cpu, driver: "llvmpipe", driver_info: "Mesa 23.2.1 refresh - kisak-mesa PPA (LLVM 15.0.7)", backend: Vulkan }
ALSA lib confmisc.c:855:(parse_card) cannot find card '0'
ALSA lib conf.c:5178:(_snd_config_evaluate) function snd_func_card_inum returned error: No such file or directory
ALSA lib confmisc.c:422:(snd_func_concat) error evaluating strings
ALSA lib conf.c:5178:(_snd_config_evaluate) function snd_func_concat returned error: No such file or directory
ALSA lib confmisc.c:1334:(snd_func_refer) error evaluating name
ALSA lib conf.c:5178:(_snd_config_evaluate) function snd_func_refer returned error: No such file or directory
ALSA lib conf.c:5701:(snd_config_expand) Evaluate error: No such file or directory
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM default
[2m2023-11-20T18:58:57.606329Z[0m [33m WARN[0m [2mbevy_audio::audio_output[0m[2m:[0m No audio device found.
[2m2023-11-20T18:58:57.876034Z[0m [32m INFO[0m [2mbevy_diagnostic::system_information_diagnostics_plugin::internal[0m[2m:[0m SystemInfo { os: "Linux 22.04 Ubuntu", kernel: "6.2.0-1016-azure", cpu: "AMD EPYC 7763 64-Core Processor", core_count: "2", memory: "15.6 GiB" }
thread 'main' panicked at /home/runner/work/bevy-example-runner/bevy-example-runner/crates/bevy_asset/src/assets.rs:149:13:
internal error: entered unreachable code: entries should always be valid after a flush
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Encountered a panic in exclusive system `bevy_asset::server::handle_internal_asset_events`!
Encountered a panic in system `bevy_app::main_schedule::Main::run_main`!
